standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -602,6 +602,13 @@ public static ConfVars getMetaConf(String name) {
     CONNECTION_USER_NAME("javax.jdo.option.ConnectionUserName",
         "javax.jdo.option.ConnectionUserName", "APP",
         "Username to use against metastore database"),
+    CONNECTION_LEAK_DETECTION_THRESHOLD("javax.jdo.option.ConnectionLeakDetectionThreshold",
```

Comment:
1. This impacts HiveMetaStore. Can you refer to MetaStoreConf?
2. Add a test case in TestDataSourceProviderFactory?

Reply:
"Can you refer to MetaStoreConf?" - I didn't catch you. What do you mean? Just create constant or what?